src/service/document-info-impl.js

```diff
@@ -94,6 +105,7 @@ export class DocInfo {
       pageViewId,
       linkRels,
       metaTags,
+      replaceParams,
```

Comment:
given we don't really 'replace' existing params, we should just call it `extraParams`